ios/FluentUI/Label/Label.swift

```diff
@@ -129,7 +133,7 @@ open class Label: UILabel, TokenizedControlInternal {
             return
         }
 
-        let labelFont = labelFont ?? tokenSet[.font].uiFont
+        let labelFont = tokenSet[.font].uiFont
         if maxFontSize > 0 && labelFont.pointSize > maxFontSize {
             super.font = labelFont.withSize(maxFontSize)
```

Comment:
Do we still need to set `super.font` or `super.textColor`?

Reply:
Yes because the tokenSet only holds the value, it doesn't actually set it right?

Reply:
Feels kinda of odd, but that seems to be right. Doesn't seem like we should need to set anything on super, but I'm also not sure how we'd let the system know anything changed otherwise.